### PR TITLE
Assign maintainers for newly-approved specs

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: CODEOWNERS
Comment (str4d) [inline CODEOWNERS:26]: These specs along with `https-bastion` will also have a maintainer from the Google Trust Fabric team per https://github.com/C2SP/C2SP/issues/43#issuecomment-1894178899. If that maintainer is decided before this PR is merged, I'll rebase to include them.

**Review by dconnolly — APPROVED** [review]

File: CODEOWNERS
Comment (str4d) [reply CODEOWNERS:26]: Timing out on this. A subsequent PR can add this maintainer when they are decided on.

Comment (str4d) [reply CODEOWNERS:26]: The additional maintainers are added in #57.

**Pull request merged**
